arviz/plots/elpdplot.py

```diff
@@ -22,6 +22,7 @@ def plot_elpd(
     ax=None,
     ic=None,
     scale=None,
+    marker="auto",
```

Comment:
We should try to get some common API between matplotlib and bokeh. The marker in matplotlib can be modified via `plot_kwargs`. @ahartikainen what do you think will be better, pop marker from `plot_kwargs` or integrate the new `marker` with matplotlib backend?